setup.py

```diff
@@ -27,5 +27,5 @@
           'Programming Language :: Python :: 2.7',
       ],
       keywords='graphql database compiler orientdb',
-      python_requires='>=2.7, <3',
+      python_requires='>=2.7, <3.7',
```

Comment:
I'd make this `<4` since I'd expect `3.7` won't break the few things this library depends on.